includes/amp-helper-functions.php

```diff
@@ -1351,6 +1351,26 @@ function amp_is_using_native_img() {
 	return (bool) apply_filters( 'amp_using_native_img', false );
 }
 
+/**
+ * Determine whether to allow native `POST` forms without conversion to use the `action-xhr` attribute and use the amp-form component.
+ *
+ * @since 2.2
+ * @link https://github.com/ampproject/amphtml/issues/27638
+ *
+ * @return bool Whether to allow native `POST` forms.
+ */
+function amp_is_allowing_native_post_forms() {
+	/**
+	 * Filters whether to allow native `POST` forms without conversion to use the `action-xhr` attribute and use the amp-form component.
+	 *
+	 * @since 2.2
+	 * @link https://github.com/ampproject/amphtml/issues/27638
+	 *
+	 * @param bool $use_native Whether to allow native `POST` forms.
+	 */
+	return (bool) apply_filters( 'amp_allowing_native_post_forms', false );
```

Comment:
Any reason for using `allowing` instead of `allow` (or `allows`)? Just wondering since we don't have any hooks that use present participle verbs, AFAIK.

Reply:
There's `amp_using_native_img` 😄 

If you think the filters make more sense as `amp_use_native_img` and `amp_allow_native_form` then that I'm happy with that.

Reply:
I had gone with singular for `img` since `imgs` looks bad.

Reply:
But then what about the function names?

Reply:
> If you think the filters make more sense as `amp_use_native_img` and `amp_allow_native_form` then that I'm happy with that.

I was only concerned with it as it doesn't follow the usual naming conventions we use for our hooks, but either naming works for me 😄.

> But then what about the function names?

They can stay if we're keeping the hook names.

Reply:
What about:

Function | Filter
---------|------
`amp_is_native_img_used()` | `amp_native_img_used`
`amp_is_native_post_form_allowed()` | `amp_native_post_form_allowed`